certbot/cli.py

```diff
@@ -699,14 +732,16 @@ def prepare_and_parse_args(plugins, args, detect_defaults=False):  # pylint: dis
         version="%(prog)s {0}".format(certbot.__version__),
         help="show program's version number and exit")
     helpful.add(
-        "automation", "--force-renewal", "--renew-by-default",
+        ["automation", "renew"],
```

Comment:
I skipped this because I feel that it's rarely the correct thing for anyone to be using...